index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link rel="stylesheet" href="css/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <h4>Eyasu</h4>

```

Comment:
- [ ] Kindly put your site logo inside an `<a>` tag instead of the `<h4>` tag for semantic reasons and when you click on the site logo it supposes to take you to the home page.

Reply:
I removed the `h4` tag and used `a` for the logo